docs/src/tutorial/add-remove-nodes.md

```diff
@@ -0,0 +1,98 @@
+# Adding and Removing Nodes
+
+## What you will need
+- A bootstrapped cluster using Canonical K8s (See (getting-started.md)[Getting Started]
+- The `dns` and `network` components enabled on your cluster.
+
+### 1. Verify the Status Of Your Cluster
+
+Verify the status of your cluster with:
+
+```
+sudo k8s status
+```
+
+Make sure you can see `k8s is ready.` in the output of the command.
+
+### 2. Start a Multipass VM
+
+Our second node will live on a Multipass VM. Let's launch it:
+
+```
+multipass launch --name expert-elf
```

Comment:
Explain what this command does something like: This command uses `multipass` to launch a virtual machine, running ubuntu 22.04, with the name `expert-elf`. 
You could add `multipass list` to verify the command was successful.

Reply:
Agreed for indicating it'll be an Ubuntu 22.04 VM. For the second part, Multipass already outputs `Launched: expert-elf`